drivers/counter/Kconfig.nxp_pcf85263a

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2022 Fraunhofer AICOS
+#
+
+config COUNTER_NXP_PCF85263A
+	bool "NXP PCF85263A RTC/Stop-watch"
+	depends on I2C
+	help
+	  Enable counter driver based on NXP PCF85263A I2C device.
+
+choice
+	prompt "PCF85263A INTA pin mode"
+
+config NXP_PCF85263A_INTA_CLK_OUT
+	bool "CLK output mode"
+
+config NXP_PCF85263A_INTA_BATT_OUT
+	bool "Battery mode indication"
+
+config NXP_PCF85263A_INTA_INT_OUT
+	bool "Interrupt (INTA) output"
+
+config NXP_PCF85263A_INTA_HIZ
+	bool "Hi-Z"
+
+endchoice
+
+
+choice
```

Comment:
Same comment here- exposing this property via devicetree allows it to be different for each instance of a PCF85263A on the board